### PR TITLE
DOC-1077: Doc new Manager role

### PR DESCRIPTION
**Pull request by lanceleonard**
This is a focused update designed to describe the new new manager role and add a table summarizing role permissions.
(Future updates will update the topic to reflect new UX.)

### CODE REVIEW

**Review by rrelledge — APPROVED** [review]
Looks good!

**Review by kaitlynmichael — APPROVED** [review]
LGTM

**Comment by lanceleonard** [comment]
The article has been updated to reflect the new UX.

A [staged preview](https://docs.redis.com/staging/jira-doc-1077/rc/administration/account-team-settings/) is available.

File: content/rc/administration/account-team-settings.md [hunk 1]
```diff
@@ -1,122 +1,142 @@
...
-- Configure Multi-Factor Authentication (MFA)
+The **Access management** screen helps you manage:
 
-When you set up [SSL/TLS]({{< relref "/rc/security/database-security/tls-ssl.md" >}}) for your account,
-you must enter the downloadable Redis CA Certificate from this page.
+- The team of users allowed to access your subscription and its databases
+- The API keys that authenticate application access to your account.
 
-![settings](/images/rc/settings.png)
+Here, you learn how to manage your team users.
```
Comment (rrelledge) [inline content/rc/administration/account-team-settings.md:17]:
```suggestion
Here, you learn how to manage your team's users.  
```

Comment (rrelledge) [inline content/rc/administration/account-team-settings.md:17]:
```suggestion
Here, you learn how to manage your team's users.  
```

**Review by rrelledge — APPROVED** [review]
The updates look good to me too.

File: content/rc/administration/account-team-settings.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/administration/account-team-settings.md]: Fixed.

**Pull request merged**